scarb/Cargo.toml

```diff
@@ -76,6 +76,12 @@ xxhash-rust.workspace = true
 zip.workspace = true
 zstd.workspace = true
 
+[target.'cfg(not(unix))'.dependencies]
+reqwest = { workspace = true, default-features = true}
+
+[target.'cfg(unix)'.dependencies]
+reqwest = { workspace = true, features = ["rustls-tls-native-roots"] }
```

Comment:
This worked well on macOS, which is also captured by `unix` cfg. Please send a PR which changes this to be Linux-specific